app/controllers/users_controller.rb

```diff
@@ -33,7 +33,7 @@ def create
     @user = User.new(user_params_with_ip)
 
     if @user.spam?
-      @user.is_spam = @user.spam?
+      @user.is_spam = true
```

Comment:
ah, I see... it makes the call every time you call `is_spam?`.
For some reason I assumed it'd get cached... Good catch.